src/frame/types.ts

```diff
@@ -95,6 +95,7 @@ export type FrameButtonMetadata =
   | {
       action?: 'post' | 'post_redirect';
       label: string;
+      postUrl?: string;
```

Comment:
Is this part of the Frame Spec? And where is this writtend?

Reply:
<img width="728" alt="image" src="https://github.com/user-attachments/assets/30fefb51-7b1e-4445-a62e-a1638bf100a0">

Per [Button Actions](https://docs.farcaster.xyz/developers/frames/spec#button-actions), we are told that a `post` should send an HTTP request to the button's `post_url`.

But as currently configured, `getFrameMetadata` does not set a `fc:frame:button:[id]:post_url` property for post actions, only tx actions.